domlab.js

```diff
@@ -0,0 +1,13 @@
+// f to c cnvertor function
+
+function calc(userInput){
+    userInput = document.getElementsByClassName('userInput').value
```

Comment:
Incorrectly getting user input, `console.log()` the `userInput` and tweak your code until you get the number the user entered to display in the console (-2)